### ISSUE TITLE
Check hyperparameter sensitivity by EnsembleESN (Ref 1, Q2)

### ISSUE THREAD

**Issue by anton-pershin** [anton-pershin/restools#2]
Referee 1 comment:

> 2. The validation set for hyperparameters tuning presented in Fig. 6 seems to be
empirical as well. The reviewer is wondering whether another draw of turbulent
trajectory would cause noticeable differences in the optimal hyperparameters. To
confirm that the conclusions are not sensitive to the specific draw of
validation turbulent trajectory, the reviewer suggests trying several different
draws and presenting the corresponding hyperparameters.
